### DISCUSSION TITLE
[Request] Manually refresh local video metadata

### DISCUSSION THREAD

**Discussion by TeamDman** [Syncplay/syncplay#444] [Ideas]
Would be nice to have an option to force syncplay to refresh the size+length of the local video file.
You can force it to by switching to another playlist entry and switching back, but it would be nicer to have the option somewhere under the menubar.

**Comment by Et0h** [comment]
Sounds like a good idea. What I've not looked into is whether this would work for all media players, or if some only report the initial duration.

**Comment by FichteFoll** [comment]
Via #362, I suggested to automatically refresh the meta data periodically, but maybe only every 10s or so or only if filesizes differ.

**Comment by Et0h** [comment]
Does anyone have any strong arguments against (or concerns about) the information being updated periodically rather than manually? It has the advantage of not needing additional UI elements which avoids clutter.

**Comment by FichteFoll** [comment]
I'm definitely in favor of automatic updates. It shouldn't be taxing enough to be noticeable (in any metric) and I definitely wouldn't want to press a button or a key to have this information updated.

**Comment by daniel-123** [comment]
One thing I'd suggest is to make it happen not every X seconds, but X seconds after completion of last scan. Slow HDDs or networked drive locations with a ton of files can take surprisingly long time to refresh. So it seems prudent to avoid possibility of getting more than single refresh process queued.

**Comment by FichteFoll** [comment]
The file path of the loaded video file should be known by the time its size is requieried, so this wouldn't require a full media directory scan.
